From 71127e5cc937878883e6021da3da337a7aa9c099 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@openadk.org>
Date: Fri, 3 Feb 2017 06:04:16 +0100
Subject: [PATCH 10229/10229] fstat: make new code aarch64 specific

The new code get's used by MIPS64 N64 and fails.
Make the new code aarch64 specific.

Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
---
 libc/sysdeps/linux/common/fstat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libc/sysdeps/linux/common/fstat.c b/libc/sysdeps/linux/common/fstat.c
index ac77eb2..c27f926 100644
--- a/libc/sysdeps/linux/common/fstat.c
+++ b/libc/sysdeps/linux/common/fstat.c
@@ -21,7 +21,7 @@ int fstat(int fd, struct stat *buf)
 }
 libc_hidden_def(fstat)
 
-#elif __WORDSIZE == 64 && defined __NR_newfstatat
+#elif __WORDSIZE == 64 && defined __NR_newfstatat && __aarch64__
 #include <fcntl.h>
 
 int fstat(int fd, struct stat *buf)
-- 
2.1.4

